RELEASE-NOTES.md

```diff
@@ -1,5 +1,48 @@
 # Prefect Release Notes
 
+## Release 2.14.16
+
+### Enhancements
+- Add the `wait_for_flow_run` method to `PrefectClient` to allow waiting for a flow run to complete — https://github.com/PrefectHQ/prefect/pull/11305
+- Add a provisioner for `Modal` push work pools — https://github.com/PrefectHQ/prefect/pull/11665
+- Add a workflow to publish OSS UI components automatically on OSS releases — https://github.com/PrefectHQ/prefect/pull/11655
+- Expose the `limit` kwarg in `serve` — https://github.com/PrefectHQ/prefect/pull/11645
```

Comment:
maybe add "to allow.."

Reply:
I don't think "allow" is the description since it's always been possible, it's just that now the option is more visible. I'm leaning towards something like "- Expose the `limit` kwarg in `serve` to increase its visibility"

Reply:
perfect! yeah I had forgotten this one